### PR TITLE
Add update actions card feature

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Add card feature to skip or install an update. There is one option for backup with 3 choices : `yes`, `no` and `ask`. Ask will display a dialog to ask to the user if they want to back up for this installation or not.

CleanShot 2023-12-21 at 14 55 43

CleanShot 2023-12-21 at 15 01 35

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by matthiasdebaat** [comment]
Could you update the text to, based on our [design website](https://design.home-assistant.io/#components/ha-dialogs):

**Create backup?**
This will create a backup before installing.
`No` `Create`

**Comment by piitaya** [comment]
> Could you update the text to, based on our [design website](https://design.home-assistant.io/#components/ha-dialogs):
> 
> **Create backup?** This will create a backup before installing. `No` `Create`

Done !

**Comment by matthiasdebaat** [comment]
Only the question mark is missing😉

File: src/translations/en.json [hunk 1]
```diff
@@ -1530,6 +1530,11 @@
         "gateway": "Gateway: {gateway}",
         "method": "Method: {method}",
         "nameservers": "Name Servers: {nameservers}"
+      },
+      "update_backup": {
+        "title": "Create backup",
```
Comment (piitaya) [inline src/translations/en.json:1535]:
```suggestion
        "title": "Create backup?",
```

**Review by bramkragten — APPROVED** [review]

**Pull request merged**